### ISSUE TITLE
Include category names in the repo

### ISSUE THREAD

**Issue by waldyrious** [fossjobs/fossjobs#29]
Searching the code, I can't find any reference to the current list of categories shown in the sidebar:

    Administrators
    Editors
    Marketing & Sales
    Designers
    Managers
    Testers
    Consultants
    Programmers
    Researchers
    Translators
    Support

There's a list in [db/jobberbase.sql](https://github.com/fossjobs/fossjobs/blob/b23b1bfaafea263037c491cd8ef93a3212df5111/db/jobberbase.sql#L58-L66):

```sql
INSERT INTO `categories` (`id`, `name`, `var_name`, `category_order`) VALUES
(1, 'Programmers', 'programmers', 7),
(2, 'Designers', 'designers', 3),
(3, 'Administrators', 'administrators', 0),
(7, 'Marketers', 'marketers', 2),
(5, 'Testers', 'testers', 5),
(6, 'Editors', 'editors', 1),
(8, 'Managers', 'managers', 4),
(9, 'Consultants', 'consultants', 6);
```

But they don't match the actual labels used on the site. "Marketers" is actually labeled "Marketing & Sales", and there are three more categories on the site that are not present in the .sql file: "Researchers", "Translators" and "Support".

Should the jobberbase.sql file be synced with the actual database used on the site? Or is there another way to resolve this issue?

ps - I'm opening this because I want to propose renaming the "Editors" category to "Writers & Editors", but found no way to contribute that change.

**Comment by waldyrious** [comment]
By the way, it's also unclear how "Administrators" differs from "Managers". Perhaps "Administrative assistants" would be clearer? Or simply "Assistants"?

**Comment by moba** [comment]
Yeah, true, the database is not currently synced. The jobberbase.sql is an initial import, and the customization happens "later". Before we figure out how to best keep it in sync it would be nice to at least do backups of the running database in the first place... ;) I now changed "Editors" to "Writers & Editors" as you proposed via the admin web panel.

I agree that it is difficult to cleanly specify categories. For me, "Managers" are responsible to manage certain aspects of the project, and "Administrators" are doing backend stuff like financial aspects, paperwork etc. -- maybe more of a more specific type of "Manager" then. "Administrators" is also confusing many since they hear "(System) Administrators", sysops.

I would not mind changing the categories, as you can see we only did very small changes from the initial set proposed by Jobberbase, but "Assistants" sounds even more generic and "low hierarchy", and for me it would not clarify anything. It will always be difficult to accommodate all the different organizational structures and preferences of FOSS projects: Many community projects would argue that there is no "management" but "Managers" are actually "Assistants dealing with the boring stuff", or "Coordinators". 

From what I can see it is most common to separate the roles dealing with the financial and legal aspects, which I would summarize as "Administrative", from everything else.

**Comment by waldyrious** [comment]
I agree with your points. "Administratives" would work well IMO, in place of "Administrators". That would make the "Manager" category non-confusing as far as I'm concerned, so it could be left as it is. Organizations who prefer emphasizing the [servant role of managers](http://www.aaronsw.com/weblog/management) can always categorize them under the Administratives category :)
